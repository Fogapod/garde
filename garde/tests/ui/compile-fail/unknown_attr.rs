#![allow(dead_code)]

#[derive(garde::Validate)]
#[garde(unknown_attr)]
struct Test {}

fn main() {}
