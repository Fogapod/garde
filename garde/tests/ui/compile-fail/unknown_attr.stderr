error: unrecognized attribute
 --> tests/ui/compile-fail/unknown_attr.rs
  |
  | #[garde(unknown_attr)]
  |         ^^^^^^^^^^^^
